thirdparty/jquery-entwine/jquery.entwine.js

```diff
@@ -42,6 +42,9 @@ var Base;
 
 	Base.addMethods = function(props) {
 		for (var name in props) {
+			if (!props.hasOwnProperty(name)) {
```

Comment:
This has been added quite a few times as best practice and to fix a regression where sometimes it tries to include `prototype` in the loop. We only ever want to loop over the properties that are being added by our code, not built-ins.